typescript/infra/config/environments/mainnet3/warp/configGetters/getBlastZeroNetworkUSDBWarpConfig.ts

```diff
@@ -0,0 +1,42 @@
+import {
+  ChainMap,
+  IsmType,
+  RouterConfig,
+  TokenRouterConfig,
+  TokenType,
+} from '@hyperlane-xyz/sdk';
+
+import { safes } from '../../owners.js';
+
+const USDB_ADDRESS = '0x4300000000000000000000000000000000000003';
+
+export const getBlastZeroNetworkUSDBWarpConfig = async (
+  routerConfig: ChainMap<RouterConfig>,
+): Promise<ChainMap<TokenRouterConfig>> => {
+  const blast: TokenRouterConfig = {
+    ...routerConfig.blast,
+    type: TokenType.collateral,
+    token: USDB_ADDRESS,
+    interchainSecurityModule: {
+      owner: '0x723e7694dc346e5a15fB6F6A0144479aC624C66F',
+      type: IsmType.FALLBACK_ROUTING,
+      domains: {},
+    },
+    owner: safes.blast,
```

Comment:
we should remove all explicit `owner` setting and any use of the `safes` object directly - if you console.log `routerConfig.blast`, you'll see that the owner is already set as part of `routerConfig.blast`

The reason is that we don't always want Safe ownership, sometimes we want ICA ownership. E.g. on scroll, we actually want our ICA on there to own this https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/main/typescript/infra/config/environments/mainnet3/owners.ts#L118

The infra checker's logic says "if the config asks for an owner that is equal to the environment's intended owner address (which is sometimes a Safe, sometimes an ICA), then make sure that the proxy admin that's used is the same one for the rest of the environment (i.e. the one from addresses.yaml. Otherwise, if a different owner is desired, keep the existing proxy admin and transfer ownership of that proxy admin to your different desired owner"

This is why you're seeing the different behavior on scroll - the desired owner for the mainnet environment on scroll is the ICA, but you're setting it to the Safe